.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Run flake8
         run: flake8 app/
 
-      - name: Run tests
+      - name: Run test
```

Comment:
why did you do this?

Reply:
I thought that it will fix my problem, but indeed it is not:)